lib/red_blocks/version.rb

```diff
@@ -1,3 +1,3 @@
 module RedBlocks
-  VERSION = "0.1.6"
+  VERSION = "0.1.7"
```

Comment:
I confirm that there is no 0.1.6 left
<img width="1077" alt="image" src="https://github.com/Altech/red_blocks/assets/53637056/619fd773-4450-4c06-ad54-9e156e82dc8b">